Tests/create_entities_for_nightly_sdk.py

```diff
@@ -0,0 +1,135 @@
+import re
+import subprocess
+from pathlib import Path
+from typing import Tuple, Union
+
+
+def run_command(cmd: str) -> Tuple[str, str]:
+    return subprocess.Popen(cmd.split(), stdout=subprocess.PIPE, stderr=subprocess.PIPE, encoding='utf-8').communicate()
+
+
+def create_incident_field(path: Path, layout_name: str) -> str:
+    """
+    Creates an incident field
+
+    Args:
+        path: A path of the pack
+        layout_name: a layout to associate the incident field
+
+    Returns:
+        The path to the incident field
+    """
+    incident_field_path = path / 'IncidentFields'
+    hello_field_path = incident_field_path / 'incidentfield-Hello_World_Type.json'
+    with open(hello_field_path) as stream:
+        field = json.load(stream)
+    name = 'Hello World Incident Test'
+    cliname = name.lower().replace(' ', '')
+    field.update({
+        'name': name,
+        'cliName': cliname,
+        'id': f'incident_{cliname}',
+        'layout': layout_name
+    })
+    field_path = path / 'IncidentFields' / f'incidentfield-{name.replace(" ", "_")}.json'
+    with open(field_path, 'w+') as stream:
+        json.dump(field, stream, indent=4)
+    return str(field_path)
+
+
+def create_layout(path: Path, layout_name: str) -> str:
+    """
+    Creates a layout field
+
+    Args:
+        path: A path of the pack
+        layout_name: a layout name to create
+
+    Returns:
+        The path to the layout
+    """
+    layout_path = path / 'Layouts'
+    layout_path_sample = layout_path / 'layout-details-Hello_World_Alert-V2.json'
+    with open(layout_path_sample) as stream:
+        layout = json.load(stream)
+    layout.update({
+        'TypeName': layout_name,
+        'typeId': layout_name
+    })
+    layout['layout'].update({
+        'id': layout_name,
+        'typeId': layout_name
+    })
+    layout_path = path / 'Layouts' / f'layout-{layout_name.replace(" ", "_")}.json'
+    with open(layout_path, 'w+') as stream:
+        json.dump(layout, stream, indent=4)
+    return str(layout_path)
+
+
+def create_incident_type(path: Path, layout_name: str) -> str:
+    """
+    Creates an incident type
+
+    Args:
+        path: A path of the pack
+        layout_name: a layout to associate the incident field
+
+    Returns:
+        The path to the incident type
+    """
+    incident_type_path = path / 'IncidentTypes'
+    incident_type_path_sample = incident_type_path / 'incidenttype-Hello_World_Alert.json'
+    with open(incident_type_path_sample) as stream:
+        incident_type = json.load(stream)
+    name = 'Hello World Alert Test'
+    incident_type.update({
+        'name': name,
+        'id': name,
+        'layout': layout_name
+    })
+    incident_path = incident_type_path / f'incidenttype-{name.replace(" ", "_")}.json'
+    with open(incident_path, 'w+') as stream:
+        json.dump(incident_type, stream, indent=4)
+    return str(incident_path)
+
+
+def upload_to_sdk(path: Union[Path, str], *args: str):
```

Comment:
why is this part of the script and not run after it?

Reply:
Bug I tried to workaround.
Trying without it